src/frontend/planner_test/tests/testdata/output/agg.yaml

```diff
@@ -1,40 +1,40 @@
 # This file is automatically generated. See `src/frontend/planner_test/README.md` for more information.
 - sql: |
     values(sum(1));
-  binder_error: |-
-    Bind error: failed to bind expression: sum(1)
+  binder_error: |
+    Failed to bind expression: sum(1)
 
-    Caused by:
-      Invalid input syntax: aggregate functions are not allowed in VALUES
+    Caused by this error:
+      1: Invalid input syntax: aggregate functions are not allowed in VALUES
```

Comment:
Is this a little verbose? 🤪 (`this error` and `1:`)

Reply:
Copied from `snafu`. I'm okay to change that.

Reply:
Welcome for pull requests. 🤣